PowerEditor/installer/nativeLang/german.xml

```diff
@@ -1369,6 +1381,7 @@ Finde in allen Dateien außer exe, obj &amp;&amp; log:
 			<finder-select-all value="Alles wählen"/>
 			<finder-clear-all value="Alles löschen"/>
 			<finder-open-all value="Alles öffnen"/>
+			<finder-purge-for-every-search value="Für jede Suche neu eingeben"/>
```

Comment:
@schnurlos
Hmm I think this translation could be misleading. Maybe it is better to write: "Vor jeder Suche leeren" or something similar?

Reply:
@MarkusBodensee: confirmed - ich werde das anpassen, Danke für den Tip.